src/hooks/useQuizState/useQuizApiState.ts

```diff
@@ -85,7 +92,17 @@ const useQuizApiState: UseQuizApiState = (
 
   const dispatchSharedQuizResults = async () => {
     try {
-      const quizResults = await getBrowseResultsForItemIds(cioClient, queryItems);
+      let quizResults;
+      try {
+        quizResults = await getQuizResults(cioClient, quizId, {
+          answers,
+          resultsPerPage: resultsPageOptions?.numResultsToDisplay,
+          quizVersionId: quizVersionIdFromParam,
+          ...resultsPageOptions?.requestConfigs,
+        });
+      } catch (error) {
```

Comment:
Can you please explain why this catch is being in place? Just want to understand when do you expect the getQuizResults to fail

Reply:
@v-i-s-h-n-u-ps

Reply:
In case the version of the quiz is no longer available. We fall back to the original behaviour in that case.